core/tests/puppeteer-acceptance-tests/blog-admin-tests/assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -17,6 +17,10 @@
  * tag properties from the blog-admin page.
  */
 
+// TODO(#16552): user-story(B8 cell) left as blocked by this issue.
+// TODO(#17162): This test can be done without the need of a super admin, as
```

Comment:
TODOs should be written as todos, i.e. the part after the todo should explain what needs to be done and be written as an instruction.

If the instruction is to not use the super admin here then why not do it in this PR?

Reply:
this is not the case now, the only way to do so is by the superadmin now. but after the issue #17162 is fixed, there is no need of super admin here. modified the comment for better understanding